docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
```

Comment:
Not saying for sure it should be, but it seems like this would follow the convention right?

Reply:
I was thinking of putting compose spec files under a "compose" directory later, when we'll have more of them. I'd put them under "compose", because the current "docker" contains building related stuff (Dockerfiles), while "compose" would contain container orchestration related stuff. How does that sound? Also, let me know if you prefer me moving it in this PR or later is okay

Reply:
Your plan sounds fine to me. I have a slight preference for making the compose directory now though.

Reply:
Done.